ubr/conf.py

```diff
@@ -122,6 +122,10 @@ def var(envname, cfgpath, default):
 # config
 #
 
+# CLI argument parsing uses the values in this map as defaults
+# tests and other non-standard entry points should use the values in
+# this map if parsed CLI arguments are not available
+DEFAULT_CLI_OPTS = {"progress_bar": True, "prompt": False}
```

Comment:
weird convention to have the `_` in place of `-` here?

Reply:
I guess this is because of Python's argparse library that takes params like `--progress-bar` and makes them `progress_bar`.